app/models/observers/notice_observer.rb

```diff
@@ -60,6 +60,17 @@ def plan_year_update(new_model_event)
             end
         end
 
+        if new_model_event.event_key == :notify_employee_of_initial_employer_ineligibility
+          errors = plan_year.enrollment_errors
+          if(errors.include?(:eligible_to_enroll_count) || errors.include?(:non_business_owner_enrollment_count))
```

Comment:
In the original ticket 17189, its mentioned that the notice has to be triggered when an initial group fails eligibility because of either one `non-owner rule`, or `minimum participation`. So, instead of `eligible_to_enroll_count`, you need to check for `enrollment_ratio`